src/_cffi_src/openssl/ssl.py

```diff
@@ -297,6 +297,9 @@
 
 SSL_SESSION *SSL_get_session(const SSL *);
 
+SSL_SESSION *d2i_SSL_SESSION(SSL_SESSION **a, const unsigned char **pp, long length);
+int i2d_SSL_SESSION(SSL_SESSION *in, unsigned char **pp);
```

Comment:
House style is to remove the parameter names, see: https://cryptography.io/en/latest/development/c-bindings/

Reply:
Thank-you. I believe I resolved this issue. The commits are squashed to hopefully improve the commit history.